### PR TITLE
Feature/industry can see requested changes

### PR DESCRIPTION
**Pull request by hamzajaved**

### CODE REVIEW

File: app/containers/Applications/ApplicationCommentsByForm.tsx [hunk 1]
```diff
@@ -13,8 +13,10 @@ interface Props {
 
 export const ApplicationCommentsByForm: React.FunctionComponent<Props> = props => {
   const {applicationReview} = props;
-  const reviewComments =
-    applicationReview.reviewCommentsByApplicationReviewId.edges;
+  const reviewComments = applicationReview
```
Comment (matthieu-foucault) [inline app/containers/Applications/ApplicationCommentsByForm.tsx]: You could use optional chaining here: 
```suggestion
  const reviewComments = applicationReview?.reviewCommentsByApplicationReviewId?.edges;
```

File: app/containers/Applications/RequestedChangesByFormResult.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  relay: RelayProp;
+}
+
+export const RequestedChangesByFormResult: React.FunctionComponent<Props> = props => {
+  const formResults = props.query.application.formResultsByApplicationId.edges;
+  return (
+    <>
+      {formResults.map(({node}) => {
+        const applicationReviews = node.requestedChanges.edges;
+        if (applicationReviews.length === 0) return <></>;
```
Comment (matthieu-foucault) [inline app/containers/Applications/RequestedChangesByFormResult.tsx]:
```suggestion
        if (applicationReviews.length === 0) return null;
```

File: app/containers/Applications/RequestedChangesByFormResult.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        return (
+          <div key={node.id} className="form-result-box">
+            <div>
+              <h5> {node.formJsonByFormId.name} </h5>
```
Comment (matthieu-foucault) [inline app/containers/Applications/RequestedChangesByFormResult.tsx]:
```suggestion
              <h5>{node.formJsonByFormId.name}</h5>
```

File: app/pages/view-application.tsx [hunk 4]
```diff
@@ -23,6 +24,8 @@ class ViewApplication extends Component<Props> {
         }
         ...ApplicationDetailsContainer_query
           @arguments(applicationId: $applicationId)
+        ...RequestedChangesByFormResult_query
+          @arguments(applicationId: $applicationId)
```
Comment (matthieu-foucault) [inline app/pages/view-application.tsx]: @wenzowski Instead if passing that `applicationId` argument to the fragments, should we have 

```
application(id: $applicationId) {
  ...ApplicationDetailsContainer_application
  ...RequestedChangesByFormResult_application
}
```

And change `ApplicationDetailsContainer` and `RequestedChangesByFormResult` to be fragments on `Application`

Comment (wenzowski) [reply app/pages/view-application.tsx]: ^ absolutely

Comment (hamzajaved) [reply app/pages/view-application.tsx]: this will require a refactor a bunch of views and containers. I'd rather have this under a separate refactor task

**Comment by wenzowski** [comment]
rebased

**Comment by wenzowski** [comment]
re-set back to @hamzajaved latest push

**Comment by wenzowski** [comment]
rebased on develop

**Pull request merged**